lib/message.rb

```diff
@@ -0,0 +1,45 @@
+require 'pg'
+require_relative 'db'
+
+class Message
+
+  attr_reader :id, :message, :date
+
+  def initialize(id:, message:, date:)
+    @id = id
+    @message = message
+    @date = date
+  end
+
+  def self.all
+    connection = Db.connect
+    result = connection.exec("SELECT * FROM peeps")
+    result.map do |message|
+      Message.new(id: message['id'], message: message['message'], date: message['date'])
+    end
+     
+  end
+
+  def self.create(message:) 
+    connection = Db.connect
+    connection.exec_params("INSERT INTO peeps (message,date) VALUES($1,current_timestamp);",[message])
```

Comment:
I wonder if you can find a neat way of pushing all the SQL into your Db class....